next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
Was this change intentional? I'm not sure if it matters either way but I remember a note (I think) about not modifying this file

Reply:
I'll revert that.  It always ends up in my builds.

Reply:
Same.. no idea why and it's kind of annoying

Reply:
@dsamojlenko @bryan-robitaille  --- I reverted the file ... I've been pulling it out of PRs for quite a while now.

Do ya'll end up with that file edited as well?  Thinking we should just commit it if you do.